x/concentrated-liquidity/tick.go

```diff
@@ -62,6 +62,13 @@ func (k Keeper) crossTick(ctx sdk.Context, poolId uint64, tickIndex int64) (liqu
 		return sdk.Dec{}, err
 	}
 
+	// subtract tick's fee growth outside from current fee accumulator
+	amountToUpdate := tickInfo.FeeGrowthOutside
+	err = k.subFromFeeAccumulator(ctx, poolId, amountToUpdate)
```

Comment:
Hmm I'm not sure why we would do this.

What needs to happen here is the following:
- update the tick's fee accumulator by subtracting the current tick's fee growth outside from the accumulator value ref: #3893 and https://github.com/osmosis-labs/osmosis/blob/3c5ed4e26f18169958a20ecd844161279351f3e4/x/concentrated-liquidity/tick.go#L65-L73